skyvern/client/client.py

```
@@ -1849,6 +1856,14 @@ def run_sdk_action(
         )
         return _response.data
 
+    @property
+    def workflows(self):
+        if self._workflows is None:
+            from .workflows.client import WorkflowsClient  # noqa: E402
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)